packages/front/svelte/views/admin/users/composents/TableUsers.svelte

```diff
@@ -1,19 +1,33 @@
 <script>
     import { createEventDispatcher } from "svelte";
-    import TableCell from "../../../../components/TableCell.svelte";
-    import TableHead from "../../../../components/TableHead.svelte";
+    import Alert from "../../../../dsfr/Alert.svelte";
 
     import Button from "../../../../dsfr/Button.svelte";
-    import Modal from "../../../../dsfr/Modal.svelte";
-    import Table from "../../../../dsfr/Table.svelte";
     import { capitalizeFirstLetter } from "../../../../helpers/textHelper";
+    import { action, data, modal } from "../../../../store/modal.store";
     import adminService from "../../admin.service";
+    import RemoveUserModal from "./RemoveUserModal.svelte";
 
     export let users;
+    let selectedUserId;
+
     const dispatch = createEventDispatcher();
-    const deleteUsersClicked = async user => {
-        await adminService.deleteUser(user._id);
-        dispatch("userDeleted");
+
+    const displayModal = user => {
```

Comment:
Je me demande si chaque modale ne devrait pas définir une fonction qui aurai pour but de l'invoquer (la modale), comme cette fonction mais séparer dans un autre fichier, je me dit que si on doit faire une modif ça sera plus simple d'allez dans un fichier spécifique à la modale plutôt que d'aller dans tous les fichiers pour la potentielle modale serait ouvert et où l'on pourrait avoirs des diff sur les datas passer en paramètre.

Dans le cas présent ça ne changerait probablement rien, mais pour d'autre futur modal réutilisable, ça serait pas mal et le faire pour celui-ci ferait de l'homogénéité !

Reply:
Je ne vois pas bien l'idée !